src/hipscat_import/index/map_reduce.py

```diff
@@ -36,49 +58,20 @@ def create_index(args):
         data["Dir"] = data["Dir"].astype(np.uint64)
         data["Npix"] = data["Npix"].astype(np.uint64)
 
-    # There are some silly dask things happening here:
-    # - Turn the existing index column into a regular column
-    # - If that had been the _hipscat_index, and we don't want it anymore, drop it
-    # - Create a new index, using our target indexing_column.
-    #   Use division hints if provided.
-    data = data.reset_index()
-    if not args.include_hipscat_index:
-        data = data.drop(columns=[HIPSCAT_ID_COLUMN])
-
     if args.division_hints is not None and len(args.division_hints) > 2:
         data = data.set_index(args.indexing_column, divisions=args.division_hints)
     else:
         # Try to avoid this! It's expensive! See:
         # https://docs.dask.org/en/latest/generated/dask.dataframe.DataFrame.set_index.html
         data = data.set_index(args.indexing_column)
 
-    if args.drop_duplicates:
-        # More dask things:
-        # - Repartition the whole dataset to account for limited memory in
-        #   pyarrow in the drop_duplicates implementation (
-        #   "array cannot contain more than 2147483646 bytes")
-        # - Reset the index, so the indexing_column values can be considered
-        #   when de-duping.
-        # - Drop duplicate rows
-        # - Set the index back to our indexing_column, but this time, the
-        #   values are still sorted so it's cheaper.
-        data = (
-            data.repartition(partition_size=1_000_000_000)
-            .reset_index()
-            .drop_duplicates()
-            .set_index(args.indexing_column, sorted=True, partition_size=args.compute_partition_size)
-        )
-    else:
-        data = data.repartition(partition_size=args.compute_partition_size)
+    data = data.repartition(partition_size=args.compute_partition_size)
```

Comment:
These changes are awesome! I'm not sure we need this line as we're passing the partition size in the `compute_kwargs` below, but I'm assuming it shouldn't be a big deal (?)